public/video-ui/src/components/FormFields/ScribeEditor.js

```diff
@@ -184,29 +158,37 @@ export class ScribeEditor extends React.Component {
   }
 
   configureScribe() {
+
+    const allKeyboardShortcuts = {
+      bold: function(event) {
+        return event.metaKey && event.keyCode === 66;
```

Comment:
could we add these [here](https://github.com/guardian/media-atom-maker/blob/master/public/video-ui/src/constants/keyCodes.js)?